### PR TITLE
ch01-02-hello-world.md partial spanish translation

### PR DESCRIPTION
**Pull request by Osovich**
Partial translation of hello-world, around 1100 words to spanish. It used to be from line 1 to 214 but now the file is a little different and i had to skip some sections, the translations end at line 267.

### CODE REVIEW

**Comment by Osovich** [comment]
Updated, now it's translated from line 1 to line 267 completely, no untranslated text in-between.

File: second-edition/src/ch01-00-introduction.md [hunk 1]
```diff
@@ -1,126 +1,127 @@
...
+Rust es para las personas que quieren construir el lenguaje de programación Rust,
+la comunidad, las herramientas de desarrollo, y las librerias. Nos encantaría
+que nos ayudaras con el lenguaje Rust.
+
+### Personas Que Valoran La Velocidad y La Estabilidad
+
+Por velocidad, nos referimos a la velocidad de los programas que Rust le permite
+crear y la velocidad con la cual Rust le permite escribirlos. Las comprobaciones
+del compilador de Rust aseguran estabilidad a través de la añadición de
+características y la refactorización, a diferencia del legado de códigos
```
Comment (thecodix) [inline second-edition/src/ch01-00-introduction.md]: 'legado de codigos quebrantables', I would suggest to translate by 'código heredado' or some variation, which is a more precise way to define legacy code

Comment (Osovich) [reply second-edition/src/ch01-00-introduction.md]: will do right away!

**Comment by thecodix** [comment]
machine translated

**Pull request merged**